observability/logging/logging-6.0/6x-cluster-logging-deploying-6.0.adoc

```diff
@@ -0,0 +1,60 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[id="installing-logging-6-0"]
+= Installing Logging
+:context: installing-logging-6-0
+
+toc::[]
+
+{product-title} Operators use custom resources (CRs) to manage applications and their components. You provide high-level configuration and settings through the CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the logic of the Operator. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs to generate CRs.
+
+
+To get started with {logging}, you must install the following Operators:
+
+* {loki-op} to manage your log store.
+* {clo} to manage log collection and forwarding.
+* {coo-first} to manage visualization.
+
+You can use either the {product-title} web console or the {product-title} CLI to install or configure {logging}.
+
+[IMPORTANT]
+====
+You must configure the {clo} after the {loki-op}.
+====
+
+
+ifdef::openshift-origin[]
+[id="prerequisites_cluster-logging-deploying_{context}"]
+== Prerequisites
+* You have downloaded the {cluster-manager-url-pull} as shown in "Obtaining the installation program" in the installation documentation for your platform.
++
+If you have the pull secret, add the `redhat-operators` catalog to the `OperatorHub` custom resource (CR) as shown in "Configuring {product-title} to use Red{nbsp}Hat Operators".
+endif::[]
+
+[id="installing-loki-and-logging-cli_{context}"]
+== Installation by using the CLI
+
+The following sections describe installing the {loki-op} and the {clo} by using the CLI.
+
+include::modules/log6x-logging-loki-cli-install.adoc[leveloffset=+2]
+include::modules/log6x-logging-clo-cli-install.adoc[leveloffset=+2]
+
+[id="installing-loki-and-logging-gui_{context}"]
+== Installation by using the web console
+
+The following sections describe installing the {loki-op} and the {clo} by using the web console.
+
+include::modules/log6x-logging-loki-gui-install.adoc[leveloffset=+2]
+include::modules/log6x-logging-clo-gui-install.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+ifdef::openshift-enterprise,openshift-origin[]
+* xref:../../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#ovn-k-network-policy[About OVN-Kubernetes network policy]
+endif::[]
+ifdef::openshift-rosa,openshift-dedicated[]
+* link:https://docs.openshift.com/container-platform/latest/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html[About the OVN-Kubernetes default Container Network Interface (CNI) network provider]
```

Comment:
Not to question you, just curious. Why the hard link here? Do we want to update it to docs.redhat?